src/v/storage/probe.cc

```diff
@@ -152,6 +152,11 @@ void probe::setup_metrics(const model::ntp& ntp) {
           [this] { return _partition_bytes; },
           sm::description("Current size of partition in bytes"),
           labels),
+        sm::make_counter(
```

Comment:
nit: prometheus convention suggests to suffix counters with `_total` and unit (https://prometheus.io/docs/practices/naming/)

`prefix_trunacted_bytes_total`

Reply:
Awesome. I was just copying other metrics in the tree, but I'll update the naming. No reason to perpetuate a poor design choice.

Reply:
i updated the naming to include units. all these metrics are gauges, so i think total as a suffix may not match the suggested naming convention.